client/src/pages/Exams.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useApi } from "../hooks/use-api";
+export default function Home() {
+  //useEffect
+  //render exams
+  const { response } = useApi();
```

Comment:
so here we would want to do something like this instead of using useApi(); as it will be confusing and is only 1 use as i understand it, useApi would just be solely for the localhost as I described above

upon project implementation we would do, ${root}/patients or something like that

This is code that worked for me

```js
import React, { useState, useEffect } from "react";

const ExamsPage = () => {

    const [exams, setExams] = useState([]);

    useEffect(() => {
        const fetchExams = async () => {
            try {
                const response = await fetch('https://czi-covid-lypkrzry4q-uc.a.run.app/api/exams')
                const res = await response.json();
                const exams_data = res["exams"] // or res.exams
                setExams(exams_data)
            } catch (e) {
                console.error(e)
            }
        }
        fetchExams();
    }, []);

    return (
        <>
            <h2>Patient Exams</h2>
            <h1>List All Exams</h1>
            {exams.map(exam => (
                exam && (
                    <>
                        <div key={exam._id}>
                            {exam.patientId}
                        </div>
                    </>
                )
            ))}
        </>
    );
}

export default ExamsPage;
```

I do believe there can be a better way to do above with using react fragments, I need to figure that out

Reply:
Im happy to see this because this is EXACTLY how I reformatted the issue

Reply:
I feel like this isn’t the best final version
React fragments are not really recommended

Reply:
using a exam info component and passing exam as prop